chrome/background.js

```diff
@@ -95,6 +95,42 @@ function onMessage(request, sender, sendResponse) {
       localStorage.getItem("use_fuzzy_search") != "false";
     sendResponse({ use_fuzzy_search: use_fuzzy_search });
   }
+
+  // spawn a new tab with pre-provided credentials
+  if (request.action == "launch") {
+    chrome.tabs.create({url: request.url}, function (tab) {
+      var authAttempted = false;
+      chrome.webRequest.onAuthRequired.addListener(
+        function authListener(requestDetails) {
+          // only supply credentials if this is the first time for this tab
+          if (!authAttempted) {
```

Comment:
could you invert this too, i.e. `if (authAttempted) { return {}; }`? Also judging by line 123 returning `{}` is important in this case too.

Reply:
No problem - done :-).